02_getting_started.asciidoc

```diff
@@ -173,21 +173,21 @@ image::images/eclair-playstore.png["Eclair wallet in the Google Play Store"]
 
 [TIP]
 ====
-It is possible to experiment and test all Bitcoin-type software with zero risk (except for your own time) by using Testnet bitcoins. You can also download Eclair Testnet wallet to try lightning (on Testnet) by going to the Google Play Store.
+It is possible to experiment and test all Bitcoin-type software with zero risk (except for your own time) by using testnet bitcoins. You can also download Eclair testnet wallet to try Lightning (on testnet) by going to the Google Play Store.
 ====
 
-Alice notices a few different elements on this page, that help her ascertain that this is, most likely, the correct "Eclair Mobile" wallet she is looking for. Firstly, the organization "ACINQ" footnote:[ACINQ: Developers of the Eclair Mobile Lightning wallet (https://acinq.co/).] is listed as the developer of this mobile wallet, which Alice knows from her research is the correct developer. Secondly, the wallet has been installed "10,000+" times and has more than 320 positive reviews. It is unlikely this is a rogue app that has snuck into the Play Store. As a third step, she goes to the ACINQ website (https://acinq.co/). She verifies that the webpage is secure by checking that the address begins with HTTPS, or prefixed by a padlock in some browsers. On the website she goes to the Download section or looks for the link to the Google App store. She finds the link and clicks it. She compares that this link brings her to the very same app in the Google App Store. Satisfied by these findings, Alice installs the Eclair app on her mobile device.
+Alice notices a few different elements on this page, that help her ascertain that this is, most likely, the correct "Eclair Mobile" wallet she is looking for. Firstly, the organization "ACINQ" footnote:[ACINQ: Developers of the Eclair Mobile Lightning wallet (https://acinq.co/).] is listed as the developer of this mobile wallet, which Alice knows from her research is the correct developer. Secondly, the wallet has been installed "10,000+" times and has more than 320 positive reviews. It is unlikely this is a rogue app that has snuck into the Google Play Store. As a third step, she goes to the ACINQ website (https://acinq.co/). She verifies that the webpage is secure by checking that the address begins with https, or prefixed by a padlock in some browsers. On the website she goes to the Download section or looks for the link to the Google App store. She finds the link and clicks it. She compares that this link brings her to the very same app in the Google App Store. Satisfied by these findings, Alice installs the Eclair app on her mobile device.
```

Comment:
Note: I tried to follow this ACINQ route, but I couldn't find the download section or any reference to Eclair Mobile, only to the Phoenix wallet.

Reply:
that is good to know!